radicle/src/storage/refs.rs

```diff
@@ -282,7 +288,7 @@ impl SignedRefs<Verified> {
             }
         }
 
-        let sigref = format!("refs/remotes/{remote}/{sigref}");
+        let sigref = sigref.add_namespace(remote.into());
```

Comment:
Meta: another function name I think could be improved: `add` sounds like it's adding it to the end, I'd propose `with_namespace(remote)` or `namespaced(remote)`